releasenotes/README.md

```diff
@@ -25,14 +25,44 @@ Additional documentation:
+    - Stream results to a file with the `--stream-to-output` flag
+    - Run `sf api request rest --help` to see more examples
+
+* CHANGE: We shipped `decomposeCustomLabelsBeta`, got feedback (thank you!), and decided to change how it behaves. You can see the design proposal [here](https://github.com/forcedotcom/cli/discussions/2948). It removes an extra folder layer that other decomposition strategies needed and supports more flexible folder structures.
+Now there's `decomposeCustomLabeltsBeta2`. If you haven't used it yet, you can try it via `sf project convert source-behavior --behavior decomposeCustomLabelsBeta2`. This works if your project isn't using a preset OR if you started using the original flavor of `decomposeCustomLabeltsBeta`. Please keep the feedback coming! (source-deploy-retrieve PR [#1392](https://github.com/forcedotcom/source-deploy-retrieve/pull/1392))
+
+* FIX: Salesforce DX projects now support the GenAiFunction [metadata type](https://github.com/forcedotcom/source-deploy-retrieve/blob/main/src/registry/metadataRegistry.json) (source-deploy-retrieve PR [#1404](https://github.com/forcedotcom/source-deploy-retrieve/pull/1404), plugin-deploy-retrieve PR [#1137](https://github.com/salesforcecli/plugin-deploy-retrieve/pull/1137))
+
+* NEW: The `apex run test` and `apex get test` commands now have a new `--concise` flag
+
+Setting this flag for `apex run test` or `apex get test` will suppress passing test results and the code coverage table. Only failing tests and the summary table will be displayed. (plugin-apex PR [#504](https://github.com/salesforcecli/plugin-apex/pull/504), Github Issue [#243](https://github.com/forcedotcom/salesforcedx-apex/issues/243), Github Discussion [#2872](https://github.com/forcedotcom/cli/discussions/2872))
+
+Many thanks to [Kyle Capehart](https://github.com/k-capehart) for contributing this useful new flag!
+
+* NEW: The `sf project convert source-behavior` command will now warn if you have uncommitted work before doing project modifications. Note that this command puts components in a newly created `main/default` folder in each package directory. You might need to re-organize them into your preferred structure. (plugin-deploy-retrieve PR [#1130](https://github.com/salesforcecli/plugin-deploy-retrieve/pull/1130))
+
+* NEW: Check out the improved human readable output on the `org create scratch` and `org resume scratch` commands! They now take advantage of the new [@oclif/multi-stage-output](https://github.com/oclif/multi-stage-output) repo. Keep your eyes out for more multi-staged output improvements in the near future. (plugin-org PR [#1120](https://github.com/salesforcecli/plugin-org/pull/1120))
+
+* FIX: We have updated our scripts for generating our supported metadata types. Our metadata coverage has increased from 577 to 598. (source-deploy-retrieve PR [#1398](https://github.com/forcedotcom/source-deploy-retrieve/pull/1398))
```

Comment:
I don't think they need to know about the scripts, but do need to know about the new types (we've typically listed them out in the past)
```suggestion
* FIX: Support for 21 new metadata types (source-deploy-retrieve PR [#1398](https://github.com/forcedotcom/source-deploy-retrieve/pull/1398))
  * AssessmentConfiguration
  * ContextUseCaseMapping
  * ContractType
  * DocumentTemplate
  * EmployeeDataSyncProfile
  * ExternalAuthIdentityProvider
  * ExternalDataTranObject
  * ExternalDocStorageConfig
  * GenAiPlugin
  * LearningAchievementConfig
  * MktDataConnection
  * MktDataConnectionSrcParam
  * PortalDelegablePermissionSet
  * ProductAttrDisplayConfig
  * ProductSpecificationRecType
  * ProductSpecificationType
  * RecAlrtDataSrcExpSetDef
  * ReferencedDashboard
  * RelatedRecordAssocCriteria
  * VirtualVisitConfig
  * WaveAnalyticAssetCollection

```